aggregator/flush_mgr_options.go

```diff
@@ -35,7 +37,11 @@ const (
 )
 
 var (
-	defaultWorkerPoolSize = int(math.Max(float64(runtime.NumCPU()/8), 1.0))
+	defaultWorkerPoolSize         = int(math.Max(float64(runtime.NumCPU()/8), 1.0))
+	defaultFlushTimesKeyFormat    = "/shardset/%s/flush"
+	defaultFlushTimesPersistEvery = 10 * time.Second
+	defaultMaxNoFlushDuration     = 15 * time.Minute
```

Comment:
`maxNoFlushDuration` is how long the follower keeps metrics in memory if it fails to observe an flush timestamp update from kv. If the leader goes down, the follower will automatically be promoted to follower so that doesn't apply here. This is an extra measure put in place in case the leader is up but not persisting flush times (e.g., etcd is down, or a bug in the code), in which case the follower will hold on to the metrics in the meantime until either it receives a kv update, or 15 minutes has passed with no kv update, in which case the follower will flush the oldest 10s of data. Ideally this value should be derived from the % of memory reserved for holding metrics in memory for this particular scenario. Additionally, alerts should be set up if the duration without flushes exceeds e.g., 30s if the leader is supposed to persist the times to kv every 10s.